chunky/src/res/se/llbit/chunky/ui/render/AdvancedTab.fxml

```diff
@@ -39,6 +39,7 @@
         </HBox>
         <IntegerAdjuster fx:id="gridSize" />
         <CheckBox fx:id="preventNormalEmitterWithSampling" mnemonicParsing="false" text="Prevent normal emitter when using emitter sampling" />
+        <CheckBox fx:id="useHeightmapData" mnemonicParsing="false" text="Use Heightmap Data" />
```

Comment:
```suggestion
        <CheckBox fx:id="useHeightmapData" mnemonicParsing="false" text="Use Heightmap data for faster chunk loading" />
```

We need a way to add some longer explanation for more advanced options. But that's out of scope for this PR I guess.